### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by MarkGibbons**
### Description

Merge the PR from @kitforbes. Update the version, circleci and README formatting.

### Issues Resolved
 
[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

---
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/line/155)

### CODE REVIEW

**Comment by MarkGibbons** [comment]
I'm getting: 

#!/bin/sh -eo pipefail
# Error calling workflow: 'kitchen'
# Error calling job: 'kitchen/dokken-single'
# Unexpected argument(s): platform
# 
# -------
# Warning: This configuration was auto-generated to show you the message above.
# Don't rerun this job. Rerunning will have no effect.
false
Exited with code 1

Trying to use v2 of the sous-chefs/kitchen orb.  Any idea what's going on?

**Comment by JohnRoesler** [comment]
@MarkGibbons `platform` in the circleci config got merged with `suite` https://github.com/sous-chefs/line/blob/master/.circleci/config.yml#L16-L17 should now be https://github.com/sous-chefs/haproxy/blob/master/.circleci/config.yml#L30

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+  - delivery
+  - danger
+  - lint-yaml
+  - lint-markdown
+
 version: 2.1
 
 orbs:
-  kitchen: sous-chefs/kitchen@2
+  kitchen: sous-chefs/kitchen@2.0.2
```
Comment (damacus) [inline .circleci/config.yml]: These do the same things, `@2` should mean, use the latest of the 2 series

Comment (MarkGibbons) [reply .circleci/config.yml]: Changed back to @2.

**Comment by Raboo** [comment]
I can approve the docs, but I know zero about circle-ci stuff.

**Comment by MarkGibbons** [comment]
It's nothing new, I'm getting the circleci boiler plate to work so some other doc changes can be made.

**Review by Raboo — APPROVED** [review]

**Review by JohnRoesler — APPROVED** [review]

**Pull request merged**